src/modules/Users.pm

```diff
@@ -4561,6 +4571,9 @@ sub Write {
 	    }
 	}
     }
+
+    if (Mode->autoinst() || Mode->autoupgrade() || Mode->config()) { WriteAuthorizedKeys(); }
```

Comment:
@lslezak no, it' something I'll fix soon: `auto` is not published, so it's no accessible :(